src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -162,7 +161,9 @@ class WorkspaceInvitePage extends React.Component {
         });
         indexOffset += personalDetailsWithoutSelected.length;
 
-        if (hasUnselectedUserToInvite) {
+        const hasUnselectedUserToInvite = this.state.userToInvite && !filterText.includes(this.state.userToInvite.login);
+        const hideUserToInvite = this.state.searchTerm !== '' && !this.state.searchTerm.includes(this.state.userToInvite);
```

Comment:
@yakubbaev Do you mean `this.state.userToInvite.login`?

https://github.com/Expensify/App/assets/25520267/d2041f46-3c17-46ea-b2d1-a0bafbf7f487

Reply:
Good catch! Now I'm hiding userToInvite only if it is already included in personalDetails